src/Api/Model/Statistics/StatisticsResponse.ts

```diff
@@ -0,0 +1,16 @@
+import { ArtistFollowingInfo } from './ArtistFollowingInfo'
+import { UserInfo } from './UserInfo'
+import { ReleaseInfo } from './ReleaseInfo'
+import { ImportInfo } from './ImportInfo'
+
+export type StatisticsResponse = {
+  readonly userInfo: UserInfo
+  readonly artistFollowingInfo: ArtistFollowingInfo
+  readonly releaseInfo: ReleaseInfo
+  readonly importInfo: ImportInfo[]
+}
+
+export type ItemsPerMonth = {
+  readonly yearMonth: string
+  readonly count: number
+}
```

Comment:
This is more like what the backend sends:

```suggestion
export type ItemsPerMonth = {
  readonly [key: string]: number
}
```